src/providers/completer/environment.ts

```diff
@@ -158,21 +158,23 @@ export class Environment implements IProvider {
         }
 
         // Load environments from the package if not already done
-        if (!(pkg in this.packageEnvsAsCommand)) {
-            this.packageEnvsAsCommand[pkg] = []
+        if (!this.packageEnvsAsCommand.has(pkg)) {
+            const entry: Suggestion[] = []
+            this.packageEnvsAsCommand.set(pkg, entry)
```

Comment:
Same comment as in the other PR. I would move this instruction to line 168. It is more a matter of taste than anything else actually.